### ISSUE TITLE
Add-on simply does nothing

### ISSUE THREAD

**Issue by Jaitsu** [codebutler/firesheep#64]
Installed the add-on, and it doesn't appear to do anything, at all. No toolbar, no background process, nothing anywhere. Running Firefox  3.6.12 32-bit on Windows 7 Home Premium 64-bit.

**Comment by ebm-zz** [comment]
Firesheep doesnt seem to be compatible with the latest version of Firefox 3.6.12.  It will recognize the extension and ask permission to install it when you restart Firefox, but it doesnt install it for some reason.  The browser opens and there is no sidebar option for Firesheep.  What I ended up doing is uninstall firefox.  I went to
http://www.oldapps.com/firefox.php
and grabbed Firefox 3.6.10.  Reinstalled and it worked right away.
